test/bdd/steps/network.js

```diff
@@ -333,3 +334,36 @@ Then(/^the last import should be the same on all nodes that replicated data$/, a
     return Promise.all(promises);
 });
 
+Given(/^I remember previous import's fingerprint value and details$/, async function () {
+    expect(!!this.state.dc, 'DC node not defined. Use other step to define it.').to.be.equal(true);
+    expect(!!this.state.lastImport, 'Nothing was imported. Use other step to do it.').to.be.equal(true);
+    assert.isTrue(this.state.nodesWalletAddress !== 'null');
```

Comment:
Leave message why it failed. Imagine it crashed with message: `this.state.nodesWalletAddress !== 'null'` in a year from now.

Reply:
Maybe to stick with expect?

Reply:
done